earth_enterprise/rpms/build.gradle

```diff
@@ -62,15 +81,17 @@ if (new File('build.properties').exists()) {
     }
 }
 
-def openGeeVersionFile = new File("../src/NATIVE-DBG-x86_64/gee_long_version.txt")
+def openGeeVersionFile = new File( gee_long_version_txt )
 
-if (!openGeeVersionFile.exists()) {
+// Call scons to create the file 'gee_long_version.txt'
+// The actual parameter used by scons can remain 'release=1'
+if ( !openGeeVersionFile.exists() ) {
     GeeCommandLine.expand(
-            ["/usr/bin/scons", "release=1", "NATIVE-DBG-x86_64/gee_long_version.txt"],
-            "Runing `scons` to produce <gee_long_version.txt> failed!",
-            null, // Use default environment variables.
-            new File("../src") // Working directory.
-        )
+      ["/usr/bin/scons", "release=1", openGeeVersionFile ],
```

Comment:
I think the most straightforward solution to this is to change "release=1" here to match whatever is passed in.